int32 zone_id
int32 zone_type

geometry_msgs/Polygon polygon